tool_collections/vcftools/vcftools_compare/vcftools_compare.xml

```diff
@@ -2,7 +2,7 @@
     <description>multiple VCF datasets</description>
 
     <requirements>
-        <requirement type="package">tabix</requirement>
+        <requirement type="package" version="0.2.5">tabix</requirement>
```

Comment:
Wouldn't it be better to use:
```
        <requirement type="package" version="1.6">htslib</requirement>
```
instead? 0.2.5 is pretty ancient and tabix has been distributed as part of htslib for ages.

Reply:
I just try to fix the old package. The multicontainer builds are failing because of this. Thats also the reason I have not bumped the tool version number.
Is this ok for you?

Reply:
That's not the only problematic requirement in this tool collection:
```
tool_collections/vcftools/vcftools_annotate/vcftools_annotate.xml
5:        <requirement type="binary">echo</requirement>

tool_collections/vcftools/vcftools_isec/vcftools_isec.xml
5:        <requirement type="package">tabix</requirement>

tool_collections/vcftools/vcftools_merge/vcftools_merge.xml
5:        <requirement type="package">tabix</requirement>

tool_collections/vcftools/vcftools_slice/vcftools_slice.xml
5:        <requirement type="binary">echo</requirement>

tool_collections/vcftools/vcftools_subset/vcftools_subset.xml
5:        <requirement type="package">tabix</requirement>
```

Also, `tool_collections/vcftools/vcftools_consensus/vcftools_consensus.xml` is already using htslib to provide tabix:
```
<requirement type="package" version="1.3.2">htslib</requirement>
```

Finally, I noticed that:
- only `vcftools_consensus` cites `10.1093/bioinformatics/btr330`
- documentation links in `<help>` could be updated to https://vcftools.github.io/examples.html .

The upstream vcftools seems to be maintained, so it may be worth to use a more sustainable solution?